feature-libs/pickup-in-store/components/presentational/pickup-options/pickup-options.component.ts

```diff
@@ -89,4 +142,46 @@ export class PickupOptionsComponent implements OnChanges {
     // Return false to stop `onPickupOptionChange` being called after this
     return false;
   }
+
+  protected initializeTabs() {
+    this.tabs = [
```

Comment:
@Zeyber  here I've tried to use the `tabs$` input property here, however it didn't work for me. We can catch up on this issue on slack

Reply:
I can see there is a lot going on with this component with pre-existing change detection calls and emits. I don't think the `TabComponent` can really be designed to account for it. So, the way you update the tabs based on this component's logic is ok :)